src/OnboardingSPA/pages/Steps/GetStarted/SiteTypeSetup/SecondarySite/index.js

```diff
@@ -29,154 +29,219 @@ const StepPrimarySetup = () => {
 		setIsDrawerSuppressed( true );
 		setDrawerActiveView( VIEW_NAV_GET_STARTED );
 		setIsHeaderNavigationEnabled( true );
+		getSiteClassificationsData();
 	}, [] );
 
-	const [ clickedIndex, changeCategory ] = useState( -1 );
+	const contents = getContents();
+	const defaultPrimaryType = 'business';
```

Comment:
```suggestion
```
Should we hard-code the `business` here? We should probably make our code more robust and select the first primary category from the API response (which can also return fallback data)?